spec/features/workflow_service_creation_spec.rb

```diff
@@ -11,15 +11,17 @@
       roles: []
     )
   end
-  before do
-    allow_any_instance_of(ItemsController).to receive(:current_user)
-      .and_return(current_user)
-    allow_any_instance_of(CatalogController).to receive(:current_user)
-      .and_return(current_user)
+  before :each do
+    allow_any_instance_of(ApplicationController).to receive(:current_user).and_return(current_user)
+    @druid = 'druid:qq613vj0238' # a fixture Dor::Item record
   end
-  scenario 'redirect and display on show page' do
-    visit add_workflow_item_path 'druid:qq613vj0238'
-    click_button 'Add'
+
+  scenario 'redirect and display on show page - with JS', js: true do
```

Comment:
Hmm. Yeah that stuff can be strange some times. I think the general consensus most times is to prefer non-js specs unless js is needed for them. Maybe let's put a pin in this PR until the other PR's shake out?